config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -508,3 +508,25 @@ periodics:
       - ../kubernetes/vendor/k8s.io/apimachinery/...
       - ../kubernetes/vendor/k8s.io/apiserver/...
       - ../kubernetes/vendor/k8s.io/client-go/...
+
+- name: ci-push-kubernetes-new-golang
+  interval: 24h
+  labels:
+    preset-service-account: "true"
+    preset-dind-enabled: "true"
+  spec:
+    containers:
+      - image: gcr.io/k8s-testimages/bootstrap:v20200302-ca9526d
```

Comment:
Please check and use a newer version of bootstrap if there is one available.

Reply:
Indeed there's a newer version of bootstrap but there's a probable bug therein - when `dind` is enabled, docker service gets started and almost immediately gets stopped by `cleanup_dind` call in line 81.

FTR: [runner.sh](https://github.com/kubernetes/test-infra/blob/b5b0a3e6bf2f7aad6fb149ac6bfc6020efd7cecb/images/bootstrap/runner.sh), lines 60-84 (line 81 is where `cleanup_dind` is called and IIUC it shouldn't - it's called once again in line 113, after the job finishes) and 28-34.

Reply:
Created an appropriate issue for that [here](https://github.com/kubernetes/test-infra/issues/17412).